### ISSUE TITLE
Bug: Python events and user group calendar views are empty

### ISSUE THREAD

**Issue by JacobCoffee** [proevo/pythondotorg#2632]
### Describe the bug

Overall views and archive views are fine, but individual calendar views are now empty but show a ">>> More" pagination button.

### To Reproduce

Visit:

https://www.python.org/events/python-events/
https://www.python.org/events/python-user-group/

### Expected behavior

Not be empty

### URL to the issue

https://www.python.org/events/python-events/

### Screenshots

```bash
"![SCREENSHOT_DESCRIPTION](SCREENSHOT_LINK.png)"
```

### Browsers

Other

### Operating System

Other

**Comment by malemburg** [comment]
Only the top-level URL https://www.python.org/events/ shows entries; and this is sorted the wrong way around (I'll open a new ticket for this... https://github.com/python/pythondotorg/issues/2723).

**Comment by malemburg** [comment]
This appears to be related to the fixes for this issue: https://github.com/python/pythondotorg/issues/1701

You can see the effect using archive.org:

- Before: https://web.archive.org/web/20240929173310/https://www.python.org/events/python-events/
- After: https://web.archive.org/web/20241001122252/https://www.python.org/events/python-events/

@JacobCoffee could you please have a look, as this is really hurting the efforts we're putting into these calendars. Thanks.

**Comment by JacobCoffee** [comment]
Will look @malemburg!

**Comment by malemburg** [comment]
Ping.

**Comment by malemburg** [comment]
Thanks for fixing this, @JacobCoffee

**Comment by JacobCoffee** [comment]
sorry for the delay @malemburg!
